config/test.exs

```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+config :rollbax,
+  access_token: "123",
+  environment: Mix.env,
+  enabled: true
```

Comment:
Why did you setup configurations here (and in `dev.exs` and `config.exs`)? If you did it to make tests/`iex -S mix` work, you're likely better off using `mix test --no-start`/`iex -S mix run --no-start` so that the `:rollbax` application is not started and it doesn't try to connect to Rollbar.